src/Components/Signs/Signs.css

```diff
@@ -0,0 +1,21 @@
+main {
+  /* display: flex;
+  flex-wrap: wrap; */
+}
+.sign {
+  display: flex;
+  flex-direction: column;
```

Comment:
If you'd like you can change the flex-direction to : row and it would wrap the zodiac signs and they wouldn't be in a column to the bottom of the page.